demo/src/app/home/home.page.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, NgZone } from '@angular/core';
+import { WebView } from '@ionic-native/ionic-webview/ngx';
+
+declare var window;
+
+@Component({
+  selector: 'app-home',
+  templateUrl: 'home.page.html',
+  styleUrls: ['home.page.scss'],
+})
+export class HomePage {
+  images: Array<any> = [];
+  grid: Array<Array<string>> = [];
+  imgPerRow: number = 3;
+  options: any;
+
+  constructor(private zone: NgZone, private webview: WebView) {
+    this.requestReadPermission();
+  }
+
+  openGallery() {
+    this.options = {
+      outputType: 1,
+    };
+
+    window.imagePicker.getPictures(
+      (results) => {
+        this.images = [];
+        this.images = [];
```

Comment:
duplicate assignment
```suggestion
        this.images = [];
```

Reply:
done